src/OWML.ModHelper.Events/HarmonyHelper.cs

```diff
@@ -71,11 +78,13 @@ public void AddPostfix(MethodBase original, Type patchType, string patchMethodNa
 			Patch(original, null, postfix, null);
 		}
 
-		public void EmptyMethod<T>(string methodName) =>
-			EmptyMethod(GetMethod<T>(methodName));
+		public void EmptyMethod<T>(string methodName) => 
+			AddPrefix<T>(methodName, typeof(ClearPatch), "Prefix");
```

Comment:
this can point to the other overload, like it was previously

Reply:
done